### ISSUE TITLE
Swich mpv hook from youtube-dl to yt-dlp

### ISSUE THREAD

**Issue by kmj9** [LukeSmithxyz/voidrice#1018]
Youtube-dl is not active since july and youtube limits downloads to ~60 kb/s. 
The result is : watching youtube videos in mpv (newsboat youtube rss) is not posible (buffering) when quality is set above 360p.

yt-dlp solved download limits but there's no option like {  --ytdl-format=  } 

Solution would be to switch hook in mpv to yt-dlp.

**Comment by krisdoodle45** [comment]
I've been having this issue myself as well, but I thought it was just youtube being odd, thanks.

I fixed it very easily, you need to install the package 'youtube-dlp' via yay and add the following to '~/.config/mpv/mpv.conf':
`script-opts=ytdl_hook-ytdl_path=yt-dlp`

It might be a drop-in for youtube-dl, in which case youtube-dl could easily be replaced system-wide, but I'll test it first.

**Comment by Trebled** [comment]
Next release of mpv defaults to yt-dlp, installing alongside mpv-git in the meantime solved for me. ^_^

**Comment by kmj9** [comment]
I agree. mpv-git works fine for now till mpv upgrade to the next version and yt-dlp is sensible choice.

**Comment by TrevCan** [comment]
`yt-dlp` has been added to the official arch community repos.

I would recommend changing it now.

[package](https://archlinux.org/packages/community/any/yt-dlp/)
